src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -1,24 +1,46 @@
 package com.sprint.mission.discodeit.controller;
 
 import com.sprint.mission.discodeit.controller.api.AuthApi;
-import com.sprint.mission.discodeit.dto.Auth.LoginRequest;
 import com.sprint.mission.discodeit.dto.User.UserDto;
+import com.sprint.mission.discodeit.dto.User.UserRoleUpdateRequest;
+import com.sprint.mission.discodeit.security.DiscodeitUserDetails;
 import com.sprint.mission.discodeit.service.AuthService;
+import com.sprint.mission.discodeit.service.UserService;
 import io.swagger.v3.oas.annotations.tags.Tag;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.web.csrf.CsrfToken;
 import org.springframework.web.bind.annotation.*;
 
 
+@Slf4j
 @Tag(name = "Auth", description = "인증 API")
 @RequestMapping("api/auth")
 @RestController
 @RequiredArgsConstructor
 public class AuthController implements AuthApi {
-    private final AuthService authService;
-    @PostMapping("login")
-    public ResponseEntity<UserDto> login(@RequestBody LoginRequest loginRequest) {
-        return ResponseEntity.status(HttpStatus.OK).body(authService.login(loginRequest));
+
+    private final UserService userService;
+
+    @GetMapping("me")
+    public ResponseEntity<UserDto> getMe(@AuthenticationPrincipal DiscodeitUserDetails userDetails) {
+        return ResponseEntity.status(HttpStatus.OK).body(userDetails.getUserDto());
```

Comment:
userDetails 내 정보를 그대로 반환하면 세션 생성 시점의 정보를 반환하기에 최신화되지 않을 가능성이 있어요.  
DB 조회를 통해 항상 최신화된 정보를 내려주는게 좋을 것 같습니다.

```suggestion
        UserDto userDto = userDetails.getUserDto();
        return ResponseEntity.status(HttpStatus.OK).body(userService.findById(userDto.id()));
```